### PR TITLE
Feature/afl 56 configure environment

### PR DESCRIPTION
**Pull request by Accult**
Upgraded  to psql

### CODE REVIEW

File: coworking_project/settings.py [hunk 1]
```diff
@@ -20,12 +22,31 @@
...
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-y6o_n=jrwc1j!^$hls@rc=y4b+=pkc1b8&x)^ixz$xe35#u^7o"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
 
-ALLOWED_HOSTS = []
+
+# Завантажте змінні середовища з .env файлу
```
Comment (rdchvd) [inline coworking_project/settings.py:29]: _and that's what we call lahidna ukrainizacia_
(remove it pls)

File: pyproject.toml [hunk 2]
```diff
@@ -12,6 +12,9 @@ phonenumbers = "^8.13.22"
 django-phonenumber-field = "^7.2.0"
 pre-commit = "^3.5.0"
 django-jazzmin = "^2.6.0"
+python-dotenv = "^1.0.0"
+django-dotenv = "^1.4.2"
+psycopg2 = "^2.9.9"
```
Comment (rdchvd) [inline pyproject.toml:17]: You'd better use  `psycopg2-binary`, as I know `psycopg2` is not great for team, it can cause issues

**Review by rdchvd — CHANGES_REQUESTED** [review]
- [x]  I don't see deleted sqlite DB
- [x]  Add .env-example so users without your creds will know what to create to make the app work

File: coworking_project/settings.py (hunk 1 above)
Comment (Accult) [reply coworking_project/settings.py:29]: а чого ти доїбалась?
ми з України

**Review by rdchvd — APPROVED** [review]
Good job!!

**Review by rdchvd — APPROVED** [review]

**Pull request merged**